docs/user-guide/settings/messages-feishu.md

```
@@ -1,97 +1,100 @@
-4. 点击「**开始绑定**」→ 后端生成 device flow 凭证
-5. 弹出二维码 / 链接，去飞书里扫码授权
-6. 页面会轮询「**检查状态**」直到 `access_token` 拿到
-7. 状态变绿「已绑定」→ 自动 `start_bot`，Bot 上线
+4. 点击「**开始绑定**」→ 后端生成 device flow凭证
+5.弹出二维码 /链接，去飞书里扫码授权
+6.页面会轮询「**检查状态**」直到 `access_token`拿到
+7.状态变绿「已绑定」→ 自动 `start_bot`，Bot 上线
 
 ### 2.3 后端时序
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)